src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -62,12 +59,27 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                     .Append("id=")
                     .Append(Uri.EscapeDataString(_docId))
                     .Append("&name=")
-                    .Append(Uri.EscapeDataString(_timeseries))
-                    .Append("&from=")
-                    .Append(_from.ToString("o"))
-                    .Append("&to=")
-                    .Append(_to.ToString("o"));
-                
+                    .Append(Uri.EscapeDataString(_timeseries));
+
+                if (_ranges != null)
```

Comment:
It is valid to request without any dates?

Reply:
there was code is `TimeSeriesHandler` that treats missing 'from' parameter as `DateTime.MinValue` and missing 'to' as `DateTime.MaxValue`. 
so I assumed it is valid to request without any dates.